ROADMAP.md

```diff
@@ -23,20 +61,16 @@ Objective: "Reduce the total cost of ownership for models by minimizing the delt
     * Scale a model that is GPU bound.
     * Scale a model that is CPU bound.
 
-### High Level Work Items
-* Define the API specification (owner ellisbigelow@)
-    * Explain complete data model
-    * Document common usage patterns to meet CUJs
-
-* Implement the API specification with a CRD (owner yuzisun@)
-    * Generate a Kubebuilder CRD
-    * Define golang protos as per spec
-    * Implement ValidatingAdmissionController for API Validation
-    * Implement ReconciliationHandler to generate subresources
-
-* Integrate a KFServing component with a SeldonDeployment (owner cliveseldon@)
-    * Determine integration strategy
-    * Implement integration
-
-## Beyond Q2
-TBD
\ No newline at end of file
+## Future Unscheduled Work
+* Multi-Model Serving.
+    * Multiple KFServices share resources in the backend.
+    * GPU Sharing.
+* Flexible Inference Graphs [MLGraph CRD](https://github.com/SeldonIO/mlgraph).
+    * Model Experimentation.
+    * Ensembling.
+    * Multi Arm Bandit.
+* Skew and Outlier Detection.
+    * Online support in graph.
+    * Offline support with Payload Logging.
+* Meta-Protocol Definition.
+    * Unify disparate protocols across frameworks.
```

Comment:
Add 'Bias Detection' as one of the future work here

Reply:
Also mini-batch/adaptive batching support